### DISCUSSION TITLE
Prediction

### DISCUSSION THREAD

**Discussion by taurynamonsterxdd** [RakambdaOrg/ChannelPointsMiner#399] [Q&A] [answered]

**Comment by Rakambda** [comment, marked as answer]
Accept TOS on twitch website

**Reply by taurynamonsterxdd** [reply]
> Accept TOS on twitch website

But where?

**Reply by Rakambda** [reply]
When placing a prediction

**Reply by taurynamonsterxdd** [reply]
> When placing a prediction

?

**Reply by Rakambda** [reply]
Do a prediction manually, by yourself with your own hands, on the website, and you'll see it
